spectrocloud/schemas/cluster_namespaces.go

```diff
@@ -0,0 +1,35 @@
+package schemas
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func ClusterNamespacesSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"name": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "Name of the namespace. This is the name of the namespace in the cluster.",
+				},
+				"resource_allocation": {
+					Type:     schema.TypeMap,
+					Required: true,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
+					},
+					Description: "Resource allocation for the namespace. This is a map of resource type to resource value. For example, `{cpu_cores: '2', memory_MiB: '2048'}`",
+				},
+				"images_blacklist": {
+					Type:     schema.TypeList,
+					Optional: true,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
+					},
+					Description: "List of images to blacklist for the namespace. For example, `['nginx:latest', 'redis:latest']`",
```

Comment:
```suggestion
					Description: "List of images to disallow for the namespace. For example, `['nginx:latest', 'redis:latest']`",
```
Hmm...ideally we should avoid terms such as "whitelist" and "blacklist". Any chance we change the attribute to `allowList`